src/clj/puppetlabs/services/protocols/versioned_code.clj

```diff
@@ -5,4 +5,7 @@
 
   (current-code-id
    [this environment]
-   "Returns the current code id (representing the freshest code) for the given environment."))
+   "Returns the current code id (representing the freshest code) for the given environment.")
+  (get-code-content
+   [this environment code-id file-path]
```

Comment:
Seems like `environment` will be an optional parameter right? Should we have two versions of the service function, one with it and one without where it defaults to `nil`?